HyperPlay-Gaming/hyperplay-desktop-client
# feat(quest-page): add auto expand and tab filters

### Summary

closes https://github.com/HyperPlay-Gaming/product-management/issues/627

### How to test:

- go to the quest page
![image](https://github.com/user-attachments/assets/1bdd98be-895c-4f25-b386-8f163a813bd3)
- check quest detail is auto expanded 
![image](https://github.com/user-attachments/assets/cbb37f35-14b6-4848-b9c9-062c40e59838)


---

Use the following Checklist if you have changed something on the Backend or Frontend:
